SparkPost/2web2ui
# EO-880 View Details Page (Details Component)

### What Changed
 - Added the alert details page ( at /alerts-new/details/{id}).
 - Created the panel to list the specs of an alert on the details page.
 - Changed the links in the table and the recently triggered cards eye icon to link to this page.

### How To Test
 - Enable user account option `alerts`. This is already enabled in appteam staging.
 - See [here](https://github.com/SparkPost/2web2ui/blob/master/docs/feature-flags.md) to enable user options. Alternatively, [point your local upstreams to staging](https://confluence.int.messagesystems.com/display/ENG/Configuring+Openresty+Upstreams) and login to our appteam, CID 107 account
 - Edit your openresty configs to contain 
```
    location /api/v1/alerts {
      proxy_pass http://sp-alerts-api;
      include cors;
    }
```
under `server:80`,   `location /api/v1`
 - Navigate to `/alerts-new`.
 - Click any alert name in the table and verify that the redirected page works.
